sample_apps/dotnet/CrudAndSimpleIngestionExample.cs

```diff
@@ -391,6 +397,12 @@ public async Task WriteRecordsWithCommonAttributes()
                 WriteRecordsResponse response = await writeClient.WriteRecordsAsync(writeRecordsRequest);
                 Console.WriteLine($"Write records status code: {response.HttpStatusCode.ToString()}");
             }
+            catch (RejectedRecordsException e) {
+                Console.WriteLine("RejectedRecordsException:" + e.ToString());
+                foreach (RejectedRecord rr in e.RejectedRecords) {
+                    Console.WriteLine("RecordIndex " + rr.RecordIndex + " : " + rr.Reason);
+                }
+            }
```

Comment:
Should we have the equivalent of System.out.println("Other records were written successfully. "); here in dotnet and below in the js app?